AzureMonitorAgent/agent.py

```diff
@@ -466,6 +466,8 @@ def enable():
         if geneva_configuration and geneva_configuration.get("enable") == True:
             hutil_log_info("Detected Geneva+ mode; azuremonitoragentmgr service will be started to handle Geneva tenants")
             ensure["azuremonitoragentmgr"] = True
+            if not azure_monitor_configuration or (azure_monitor_configuration and azure_monitor_configuration.get("enable") != True):
```

Comment:
We don't need this condition; if a customer is using geneva AND amcs we still should generate the omuxsock module file